packages/runtime/test/lib/testUtils.ts

```diff
@@ -132,6 +132,32 @@ export async function syncUntilHasIdentityDeletionProcess(transportServices: Tra
     return await syncUntilHas(transportServices, "identityDeletionProcesses", (m) => m.id === identityDeletionProcessId.toString());
 }
 
+export async function resetEventBusAndSyncUntilHasEvent<TEvent extends Event>(
```

Comment:
Now that you removed the reset, the name of the functions needs to be adjusted.

Reply:
it's still there, in the first line

Reply:
I don't like blablablaANDbla. First of all you should be able to specify which even you are waiting for without resetting the eventBus, but if not you should reset it outside.

Reply:
It's possible to specify which event you are waiting for, but then it's not possible to separately check for the event arriving and the event having the correct content - don't know how relevant this is though